rpcs3/Emu/RSX/GL/GLCommonDecompiler.cpp

```diff
@@ -76,4 +76,5 @@ std::string compareFunctionImpl(COMPARE f, const std::string &Op0, const std::st
 	case COMPARE::FUNCTION_SNE:
 		return "notEqual(" + Op0 + ", " + Op1 + ")";
 	}
+	throw new EXCEPTION("Unknow compare function");
```

Comment:
Why not:
throw EXCEPTION(...);

The latter is used everywhere instead of throw new.